gizmo.py

```diff
@@ -0,0 +1,73 @@
+def hello(name,country="Finland"):
+    print("Hello {0}, how are things in {1}?".format(str(name),str(country)))
+
+def spell(input_str):
+    result=input_str[0]
+    for i in input_str[1:]:
+        result+="."+i
+        
+    return print(result)
+
+def relative_path(subject_identifiers,relpath='./subjects/moc_recording'):
+    names=[]
+    relpath='./subjects/mock_recording'
+    for i in subject_identifiers:
+        res="{0}_{1}.rec".format(relpath,i) #new style
+        #res=relpath+"_"+i+".rec" #old style, not good
+        names.append(res)
+
+    return names
+
+class Gizmo:
+    def __init__(self, name):
+        self.name=name
+
+    def speak(self):
+        print(self.name)
+
+def multiplication_table(zero_out_multiples=None):
+    """ 
+    Summary: 
+    Calculates a multiplication table based on two vectors ranging from 0 to 12 each.
```

Comment:
The first line of the docstring is taken as one line summary.

Reply:
> Good progress!

Thank you. These were the easy tasks. :-)
Re Docstring exercise: 
I could not find a good description  of the docstring, i.e. one that uses the words extended summary, summary etc. I saw the test failed. I looked into your test and I saw you seem to expect these strings. But do you accept this solution for this exercise, please? Or is there something I can improve, please?
Btw, is githhub running out of memory for the tests? Some get cancelled.